Client/Frontend/Settings/HomepageSettings/WallpaperSettings/v2/WallpaperSettingsViewController.swift

```diff
@@ -0,0 +1,285 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+class WallpaperSettingsViewController: UIViewController, Loggable {
+
+    private struct UX {
+        static let cardWidth: CGFloat = UIDevice().isTinyFormFactor ? 88 : 97
+        static let cardHeight: CGFloat = UIDevice().isTinyFormFactor ? 80 : 88
+        static let inset: CGFloat = 8
+        static let cardShadowHeight: CGFloat = 14
+        static let sectionBottomInset: CGFloat = 16
+    }
+
+    private var viewModel: WallpaperSettingsViewModel
+    internal var notificationCenter: NotificationProtocol
```

Comment:
I'm confused as to why this is `internal`. As far as I understand it, `interal` == `public` within the same module, and, given that we're not building modules, what's the preference of over the other?